src/main/java/school/lemon/changerequest/java/introduction/pr1/ConsoleCalculator.java

```diff
@@ -1,9 +1,77 @@
 package school.lemon.changerequest.java.introduction.pr1;
 
 
+import java.util.Scanner;
+
 public class ConsoleCalculator {
 
     public static void main(String[] args) {
 
+        boolean go = true;
+        Scanner userInput = new Scanner(System.in);
+        operationsChoise();
+        makeChoise();
```

Comment:
You can put `makeChoise` inside the loop, and remove from lines 26, 58. 
By the way, please rename it to `makeChoice`